src/input/element-input.js

```diff
@@ -160,19 +160,22 @@ Object.assign(pc, function () {
      * @param {TouchEvent} event The TouchEvent that was originally raised.
      * @param {pc.ElementComponent} element The ElementComponent that this event was originally raised on.
      * @param {pc.CameraComponent} camera The CameraComponent that this event was originally raised via.
-     * @param {Number} x The x coordinate of the touch that triggered the event
-     * @param {Number} y The y coordinate of the touch that triggered the event
-     * @param {pc.ElementInput} input The pc.ElementInput instance
-     * @property {pc.Touch[]} touches The Touch objects representing all current points of contact with the surface, regardless of target or changed status.
-     * @property {pc.Touch[]} changedTouches The Touch objects representing individual points of contact whose states changed between the previous touch event and this one.
+     * @param {Number} x The x coordinate of the touch that triggered the event.
+     * @param {Number} y The y coordinate of the touch that triggered the event.
+     * @param {Touch} touch The touch object that triggered the event.
+     * @param {pc.ElementInput} input The pc.ElementInput instance.
+     * @property {Touch[]} touches The Touch objects representing all current points of contact with the surface, regardless of target or changed status.
+     * @property {Touch[]} changedTouches The Touch objects representing individual points of contact whose states changed between the previous touch event and this one.
+     * @property {Touch} touch The touch object that triggered the event.
      */
-    var ElementTouchEvent = function (event, element, camera, x, y, input) {
+    var ElementTouchEvent = function (event, element, camera, x, y, touch, input) {
```

Comment:
It looks like `input` is not actually used so can you remove `input` from the constructor and update the calls that create instances of ElementTouchEvent to not pass `this` in the constructor?